code/ATMOSPHERICS/components/unary_devices/cryo.dm

```diff
@@ -80,28 +80,13 @@
 /obj/machinery/atmospherics/components/unary/cryo_cell/relaymove(mob/user)	open_machine()
 
 /obj/machinery/atmospherics/components/unary/cryo_cell/container_resist()
+	if(usr.incapacitated()) // Check that they're able to open the tube.

```

Comment:
the check isn't needed since it's already checked in https://github.com/tgstation/-tg-station/blob/8994f033775ec1073acdce9d57f487a2444eda12/code/modules/mob/living/living.dm#L665 just before calling container_resist).

Also, using incapacitated() means returning false for cuffed mob, which wouldn't be consistent with how you resist out of other containers.